src/query-builder/QueryFormatBuilder.ts

```diff
@@ -0,0 +1,120 @@
+import {EntityTarget} from "../common/EntityTarget";
+import {SelectQueryBuilder} from "./SelectQueryBuilder";
+
+/**
+ * Allows to build full sql queries 
+ */
+export interface AliasesLiteral { 
+    [aliasName: string]: EntityTarget<any> 
+};
+
+export class QueryFormatBuilder extends SelectQueryBuilder<any> {
```

Comment:
It should extend just `QueryBuilder` since it doesn't use any features from `SelectQueryBuilder`.

Reply:
Also still think `CustomQueryBuilder` would be a more accurate description.

Reply:
1) I've changed the description into CustomQueryBuilder. Looks better.
2) I've changed the extending into QueryBuilder instead SelectQueryBuilder. Functions "getRawOne" , "getRawMany" became unavailable, I had to duplicate them in CustomQueryBuilder